generate/generate.go

```diff
@@ -51,6 +51,9 @@ func Run(dir string, args ...string) error {
 	if len(pkgs) == 0 {
 		return fmt.Errorf("no Gunk packages to generate")
 	}
+	if loader.PrintErrors(pkgs) > 0 {
```

Comment:
We may want to put this in the `FileDescriptorSet` as well. We should probably try and consolidate these two functions if possible.

Reply:
True! Will take a look.

Reply:
You're completely right - I had missed this. And I agree we should join the code.

Reply:
Fixed this, added a TODO, and added a regression test for `gunk dump` to check load errors.